eng/Packages.Data.props

```diff
@@ -92,6 +92,7 @@
     <PackageReference Update="Microsoft.Azure.Amqp" Version="2.4.13" />
     <PackageReference Update="Microsoft.Identity.Client" Version="4.30.1" />
     <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="2.18.4" />
+    <PackageReference Update="Microsoft.Rest.ClientRuntime" Version="[2.3.20, 3.0.0)" />
```

Comment:
Did you get this package dependency approved? If we do we should avoid using package version ranges and instead pick the minimum version that contains what is needed.

Reply:
We generally shouldn't be referencing both Azure.Core and Microsoft.Rest.ClientRuntime.

Reply:
This package ref was not needed and neither was `Microsoft.Azure.Management.ResourceManager`. Removed both.